arroyo/processing/strategies/batching.py

```diff
@@ -113,7 +113,7 @@ def __init__(
         def generator(
             values: ValuesBatch[TStrategyPayload],
         ) -> MutableSequence[TStrategyPayload]:
-            return [value.payload for value in values]
```

Comment:
ah yeah, that's why it wasn't fixed. the types need to change then and it's a breaking change to the API. i'm fine with that

Reply:
since it's no longer the payload you're returning here, the return value needs to change from `MutableSequence[TStrategyPayload]` to `ValuesBatch`. and `[value for value in values]` is probably equivalent to just returning `values`